test/performance/performance.go

```diff
@@ -0,0 +1,381 @@
+		apiserverAvg:  avg_apiserver,
+		apiserverMax:  max_apiserver,
+	}
+}
+
+func genUniquePolicy(in_filename string, out_filename string) error {
+	input, err := ioutil.ReadFile(in_filename)
+	if err != nil {
+		return err
+	}
+
+	output := strings.ReplaceAll(string(input), "[ID]", uuid.New().String())
+
+	err = ioutil.WriteFile(out_filename, []byte(output), 0644)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+type metricData struct {
+	timestamp     string
+	numPolicies   int
+	controllerAvg float64
+	controllerMax float64
+	apiserverAvg  float64
+	apiserverMax  float64
+}
+
+// pretty print table of results to stdout
+func printTable(data []metricData) {
+	table := tabwriter.NewWriter(os.Stdout, 1, 4, 1, ' ', tabwriter.Debug|tabwriter.AlignRight)
+	fmt.Fprintln(table, "========\t==========\t====================\t====================\t===================\t===================\t")
+	fmt.Fprintln(table, "time\t# policies\tavg cpu (controller)\tmax cpu (controller)\tavg cpu (apiserver)\tmax cpu (apiserver)\t")
+	fmt.Fprintln(table, "========\t==========\t====================\t====================\t===================\t===================\t")
+
+	for i := 0; i < len(data); i++ {
+		fmt.Fprintf(table, "%s\t%d\t%s\t%s\t%s\t%s\t\n",
+			data[i].timestamp,
+			data[i].numPolicies,
+			fmt.Sprintf("%.5f", data[i].controllerAvg),
+			fmt.Sprintf("%.5f", data[i].controllerMax),
+			fmt.Sprintf("%.5f", data[i].apiserverAvg),
+			fmt.Sprintf("%.5f", data[i].apiserverMax),
+		)
+	}
+
+	table.Flush()
+}
+
+// export table of results to a csv file
+func exportTable(data []metricData, filename string) {
+	f, err := os.Create(filename)
+	if err != nil {
+		klog.Exitf("Error: failed to create %s; %s", filename, err)
+	}
+
+	defer func() {
+		if cerr := f.Close(); cerr != nil && err == nil {
+			err = cerr
+		}
+	}()
+
+	w := csv.NewWriter(f)
+	defer w.Flush()
+
+	line := []string{"time", "num_policies", "avg_cpu_controller", "max_cpu_controller", "avg_cpu_apiserver", "max_cpu_apiserver"}
+	if err := w.Write(line); err != nil {
+		klog.Exitf("Error writing headers to file; %s", err)
+	}
+
+	for _, entry := range data {
+		line = []string{
+			entry.timestamp,
+			fmt.Sprintf("%d", entry.numPolicies),
+			fmt.Sprintf("%.5f", entry.controllerAvg),
+			fmt.Sprintf("%.5f", entry.controllerMax),
+			fmt.Sprintf("%.5f", entry.apiserverAvg),
+			fmt.Sprintf("%.5f", entry.apiserverMax),
+		}
+		if err := w.Write(line); err != nil {
+			klog.Exitf("Error writing data to file; %s", err)
+		}
+	}
+}
+
+func init() {
+	klog.SetOutput(os.Stdout)
+	klog.InitFlags(flag.CommandLine)
+}
+
+func main() {
+	// pull in test variables from flags
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+
+	var plc_filename, output_filename string
+	var n_per_batch, n_total, per_batch_sleep int
+	var insecure bool
+
+	pflag.StringVarP(&plc_filename, "policy", "p",
+		"/resources/templates/cfgmap-plc.yaml", "path to test policy YAML, relative to performance directory")
```

Comment:
It's more intuitive to show a relative path as the default.
```suggestion
		"resources/templates/cfgmap-plc.yaml", "path to test policy YAML, relative to performance directory")
```